Google.PowerShell/Container/GkeClusterCmdlets.cs

```diff
@@ -0,0 +1,155 @@
+﻿// Copyright 2015-2017 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.PowerShell.Common;
+using Google.Apis.Container.v1;
+using Google.Apis.Container.v1.Data;
+using System.Management.Automation;
+using System.Collections.Generic;
+using System.Net;
+
+namespace Google.PowerShell.Container
+{
+    /// <summary>
+    /// <para type="synopsis">
+    /// Gets Google Container Clusters.
+    /// </para>
+    /// <para type="description">
+    /// Gets Google Container Clusters. If -Project parameter is not specified, the default project will be used.
+    /// If -Zone or -ClusterName is not used, the cmdlet will return every clusters in every zone in the project.
```

Comment:
"If neither -Zone nor -ClusterName is used" is better. It is too easy to skip over the "not".